docs/source/en/task_summary.mdx

```diff
@@ -10,1125 +10,224 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Summary of the tasks
+# What 🤗 Transformers can do
 
-[[open-in-colab]]
+The [Transformer](https://arxiv.org/abs/1706.03762) is a uniquely flexible and powerful architecture. While it is typically associated with natural language processing (NLP) tasks, other fields like computer vision and speech processing are increasingly adopting Transformer architectures to solve tasks in their area. If you look at some of the underlying technology behind popular consumer products like smartphones, apps, and self-driving cars, odds are that you'll find some variant of a Transformer behind it. That is because the Transformer is a general architecture that can be applied to many tasks!
```

Comment:
I wouldn't frame this around the Transformer model as we have a few models that are non-Transformer in the library and it's not clear that Transformer architectures are better than convnets in computer vision. If a non-Transformer architecture gets state-of-the-art on any task, we will include it.

So I would remove this paragraph.

Reply:
I reframed this around the 🤗 Transformers library and what it can do instead of the Transformer model.